jupyterhub_singleuser_profiles/ui/src/ImageForm/ImageForm.tsx

```diff
@@ -129,10 +129,62 @@ const ImageForm: React.FC<ImageFormProps> = ({ userConfig, onValidImage }) => {
     }
   };
 
+  const imageGroup = (imageList: ImageType[], isDefault: boolean) => (
+    <>
+      <div className="jsp-spawner__image-options__subtitle">
+        {isDefault ? 'Red Hat Maintained' : 'Custom Images'}
+      </div>
+      {isDefault ? null : (
+        <div className="jsp-spawner__image-options__subtitle-hint">
+          Manage custom images in Settings.
```

Comment:
you can remove this Manage custom images in Setttings.  We are not going to support it.

Reply:
Updated.